### DISCUSSION TITLE
Add ability to show fan speed in Thermostat card

### DISCUSSION THREAD

**Discussion by theone11** [home-assistant/frontend#19146] [Dashboard]
Is there an option to show fan speed (low/mid/high) as buttons in Thermostat card like there is for mode (heat/coo/auto/off)?

**Comment by ildar170975** [comment]
I guess Thermostat is not same as Fan.

**Reply by theone11** [reply]
But if you look in more info there is an option (bottom right) to change the fan mode
So it is there just not able to show it on card. 

![image](https://github.com/home-assistant/frontend/assets/3348953/914f7c99-b2f1-4fb0-aa40-fa2d69d98d77)

**Reply by ildar170975** [reply]
My fault! sorry.
